testing/test_assertrewrite.py

```diff
@@ -65,15 +65,30 @@ def getmsg(f, extra_ns=None, must_pass=False):
             pytest.fail("function didn't raise at all")
 
 
+def python_version_has_docstring_in_module_node():
+    """Module docstrings in 3.8 are part of Module node.
+    This was briefly in 3.7 as well but got reverted in beta 5.
+
+    TODO:
+
+    We have a complicated sys.version_info if in here to ease testing on
+    various Python 3.7 versions, but we should remove the 3.7 check after
+    3.7 is released as stable to make this check more straightforward.
+    """
+    return (
+        sys.version_info < (3, 8) or (3, 7) <= sys.version_info <= (3, 7, 0, "beta", 4)
+    )
+
+
 class TestAssertionRewrite(object):
 
     def test_place_initial_imports(self):
         s = """'Doc string'\nother = stuff"""
         m = rewrite(s)
-        # Module docstrings in 3.7 are part of Module node, it's not in the body
-        # so we remove it so the following body items have the same indexes on
-        # all Python versions
-        if sys.version_info < (3, 7):
+        # Module docstrings in some new Python versions are part of Module node
+        # It's not in the body so we remove it so the following body items have
+        # the same indexes on all Python versions:
+        if python_version_has_docstring_in_module_node():
```

Comment:
sorry for the nitpick, since in all 3 cases the following code is the same can we please factor it over as well

Reply:
Pushed in 118c30d. Feel free to squash or not squash when merging.